### ISSUE TITLE
[Tooltip]: disable manual handling of open state

### ISSUE THREAD

**Issue by rinaok** [Vonage/vivid-3#823]
### The problem

1. tooltip's open state should be handled in class implementation as specified by its [aria pattern](https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/) only
> when the element receives keyboard focus or the mouse hovers over it

2. Add description to the docs such as 
_we’re often asked for vivid tooltip to support interactive content (e.g. links).
The reason tooltip hinders any content other than plain text is to improve a11y concern which are often overlooked in development, and tooltip in that sense require a little chat.
A common pattern to avoid a11y issues -
> Elements with the tooltip role should be referenced through the use of [aria-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby) before or when the tooltip is displayed. The aria-describedby attribute is on the owning element, not on the tooltip
[MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role)

here’s a [nice summary](https://web.dev/building-a-tooltip-component/#toggletip-vs-tooltip) comparing tooltip and its extender…_

> A tooltip should contain non-interactive supplemental information

4. Remove the Popup from docs for the time being and align to its reformed name _popover_

### The solution

.

### Application/website

.

### Business priority

High Priority = pressing release

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/vonage/vivid-3/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/vonage/vivid-3/issues) for duplicate issues
